tests/TestCase/View/Helper/UrlHelperTest.php

```diff
@@ -254,6 +254,16 @@ public function testAssetUrlTimestampForce()
         $this->assertRegExp('/' . preg_quote(Configure::read('App.cssBaseUrl') . 'cake.generic.css?', '/') . '[0-9]+/', $result);
     }
 
+    public function testAssetTimestampConfigureOverride()
+    {
+        $this->Helper->webroot = '';
+        Configure::write('Asset.timestamp', 'force');
+        $timestamp = false;
+
+        $result = $this->Helper->assetTimestamp(Configure::read('App.cssBaseUrl') . 'cake.generic.css', $timestamp);
+        $this->assertEquals(Configure::read('App.cssBaseUrl') . 'cake.generic.css', $result);
+    }
```

Comment:
We should also have tests for the new option in each of the public methods as that is new API surface area we're adding.

Reply:
Fair enough! I will add those later this week.

Reply:
Awesome!